### DISCUSSION TITLE
I'm coming from ZoneMinder and I'm having trouble with Frigate fundimentals?.?

### DISCUSSION THREAD

**Discussion by farfinkooger** [blakeblackshear/frigate#15281] [Ask A Question]
Got Frigate up and running.  

1.  Can someone tell me how to "configure default logging driver" [Outlined here](https://docs.docker.com/engine/install/linux-postinstall/)
2.  Can someone point me in the direction of how to setup Amcrest cameras in Frigate AND maybe share the approach they use for the settings?  Recording settings?  Object detection settings?

Any help is appreciated.  I've wiped and installed multiple times so don't be afraid to give me a suggestion or point me in a direction.  I still have ZoneMinder running so if I break anything, I'll just wipe and install again.  Thank you for any help.

**Comment by hawkeye217** [comment]
Have you seen the official [Getting Started Guide](https://docs.frigate.video/guides/getting_started/)?

**Reply by farfinkooger** [reply]
I have.  Thank you.  

I feel like I am stuck on some of the MOST basic parts of Frigate.  :-)

**Reply by NickM-27** [reply]
It's generally easier for us to help when there are more specific questions. 

The logging driver you linked to has nothing to do with frigate as that is a docker config

There are also full docs pages for recording and object detection
